src/plugins/controls/public/options_list/components/options_list_control.tsx

```diff
@@ -168,26 +196,57 @@ export const OptionsListControl = ({
         'optionsList--filterGroupSingle': controlStyle !== 'twoLine',
       })}
     >
-      <EuiInputPopover
-        ownFocus
-        input={button}
-        hasArrow={false}
+      <EuiTourStep
```

Comment:
I've made a suggestion on how we can move this logic outside of each individual control into the control group - check it out here: https://github.com/nickpeihl/kibana/pull/3  👍 

This will not only clean up the code a bit, it will also prevent the current "storm of tour steps" we are seeing currently:

![image](https://github.com/elastic/kibana/assets/8698078/4303db1f-f7d9-420a-b665-71289590cec7)